tools/console/console.js

```diff
@@ -1325,6 +1325,15 @@ Console.prototype.readLine = function (options) {
   var previousProgressDisplay = self._progressDisplay;
   self._setProgressDisplay(new ProgressDisplayNone());
 
+  Object.getOwnPropertySymbols(process.stdin).forEach(sym => {
+    if (/^Symbol\(\w+-decoder\)$/.test(String(sym))) {
+      // Node 8.1.0 fails to attach a 'data' listener to process.stdin if
+      // it already has a StringDecoder property, so we have to remove
+      // that property if we want to use process.stdin again. #node8
+      delete process.stdin[sym];
+    }
+  });
```

Comment:
This should be rendered unnecessary by https://github.com/nodejs/node/pull/13560.